runtime/bcutil/cfreader.c

```diff
@@ -1956,6 +1956,38 @@ checkAttributes(J9CfrClassFile* classfile, J9CfrAttribute** attributes, U_32 att
 					errorCode = J9NLS_CFR_ERR_LOCAL_VARIABLE_SIGNATURE_NOT_UTF8__ID;
 					goto _errorFound;
 				}
+
+				/* Variables in the LocalVariableTypeTable will also appear in the LocalVariableTable */
+				if(flags & CFR_Xfuture) {
+					J9CfrLocalVariableTypeTableEntry lvttEntry = ((J9CfrAttributeLocalVariableTypeTable*)attrib)->localVariableTypeTable[j];
+					J9CfrAttribute* codeAttrib = NULL;
+					UDATA l, m;
+					UDATA result = FALSE;
+
+					/* There may be more than one LocalVariableTable in the code attribute's attribute table. Check each attribute to find the match */
+					for(l = 0; l < attributesCount; l++) {
+						codeAttrib = attributes[l];
+						if(codeAttrib->tag == CFR_ATTRIBUTE_LocalVariableTable) {
+							for(m = 0; m < ((J9CfrAttributeLocalVariableTable*)codeAttrib)->localVariableTableLength; m++) {
+								J9CfrLocalVariableTableEntry lvtEntry = ((J9CfrAttributeLocalVariableTable*)codeAttrib)->localVariableTable[m];
+
+								/* Check for matching entries. Signatures will not be the same. */
```

Comment:
This is not about the difference of signatures. It is because `LocalVariableTypeTable `provides `signature `information while `LocalVariableTable `provides `descriptor `information as explained in the VM Spec.

Reply:
yes. I meant that the values are not being compared because the indices are different. I will make this more clear.